WebApplication.Tests/PublisherTest.cs

```diff
@@ -0,0 +1,208 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Autodesk.Forge.Core;
+using Autodesk.Forge.DesignAutomation.Http;
+using Autodesk.Forge.DesignAutomation.Model;
+using Microsoft.Extensions.Logging.Abstractions;
+using Microsoft.Extensions.Options;
+using Moq;
+using WebApplication.Definitions;
+using WebApplication.Processing;
+using WebApplication.Utilities;
+using Xunit;
+
+namespace WebApplication.Tests
+{
+    public class PublisherTest
+    {
+        private readonly Dictionary<string, IArgument> _workItemArgs;
+        private readonly Mock<IForgeAppBase> _configMock;
+
+        public PublisherTest()
+        {
+            _workItemArgs = new Dictionary<string, IArgument>();
+
+            _configMock = new Mock<IForgeAppBase>();
+            _configMock.Setup(mock => mock.ActivityId).Returns("activityId");
+            _configMock.Setup(mock => mock.ActivityLabel).Returns("activityLabel");
+        }
+        
+        private class CallbackDataProvider : IEnumerable<object[]>
```

Comment:
I know we maybe already talked about it on another test. But why not to use simple Setup and we are using Theory and DataProvider?

Reply:
There are two different tests for callbacks. 

Moreover, in the next commit I have refactored the test and join callback and polling to the single Theory with three different sets of test data. Two for callback, one for polling.